go.mod

```diff
@@ -63,6 +64,8 @@ require (
 	google.golang.org/grpc v1.49.0
 )
 
+require github.com/celestiaorg/celestia-app v0.0.0-00010101000000-000000000000
```

Comment:
I hope this will be changed in https://github.com/celestiaorg/celestia-node/pull/1147

Reply:
This actually can be removed already so I did it